Editor/Window/HelpAndDocumentationPage.cs

```diff
@@ -36,7 +35,6 @@ private void ApplyText()
             l.SetElementText("HelpPageDescription", Strings.HelpPageDescription);
             l.SetElementText("HelpPageReportIssueLink", Strings.HelpPageReportIssueLink);
             l.SetElementText("HelpPageDocumentationLink", Strings.HelpPageDocumentationLink);
-            l.SetElementText("HelpPageVideoTutorialLink", Strings.HelpPageVideoTutorialLink);
```

Comment:
Could you clean this up in Strings.cs and TextProvider.cs as well?

Reply:
Ah yes woops, removed now.